modules/UI/UI.js

```diff
@@ -40,6 +35,13 @@ import FollowMe from "../FollowMe";
 var eventEmitter = new EventEmitter();
 UI.eventEmitter = eventEmitter;
 
+/**
+ * Whether an overlay is visible or not.
+ * @type {boolean}
+ * @public
```

Comment:
I think I'm missing something. How it can be private since it is used by another module? Do you want me to create a public setter for it? 

Or do you want to add "_" to the variable name just to mark this variable as something that shouldn't be used? If this is the case I prefer to fix this with more comments.